File: Solutionsin Haskell/p053.hs
{- 
 - Solution to Project Euler problem 53
 - Copyright (c) Sankha Saha (CODR). All rights reserved.
 - 
 - https://github.com/codr07/Project-Euler-Solusions
 -}

import qualified EulerLib


main = putStrLn (show ans)
ans = sum [1 | n <- [1..100], r <- [0..n], EulerLib.binomial n r > 10^6]

File: codes/mathematica/p053.m
(* 
 * Solution to Project Euler problem 53
 * Copyright (c) Sankha Saha. All rights reserved.
 * 
 *  
 *   
 *)


Length[Select[Flatten[Table[Binomial[n, r], {n, 1, 100}, {r, 0, n}]], Function[x, x > 1000000]]]
